views/movies/create.hbs

```diff
@@ -0,0 +1,24 @@
+<h2>Add the new movie here</h2>
+<form action="/movies" method="POST">
+  <label for="title">Title</label>
+  <input type="text" name="title" id="title">
+  <br>
+
+  <label for="genre">Genre</label>
+  <input type="text" name="genre" id="genre">
+  <br>
+
+  <label for="plot">Plot</label>
+  <input type="text" name="plot" id="plot">
+  <br>
+
+  <label for="cast">Cast</label>
+  <select name="cast" id="cast">
```

Comment:
you need the keyword "multiple" here to be able to select multiple actors.

`<select name="cast" multiple>`